addons/web/controllers/main.py

```diff
@@ -1059,6 +1059,9 @@ def content_image(self, xmlid=None, model='ir.attachment', id=None, field='datas
         image_base64 = limited_image_resize(
             image_base64, width=width, height=height, crop=crop)
 
+        if not image_base64:
+            return request.not_found()
```

Comment:
remove this commit if still useless like discuss previously

Reply:
Ok to remove (image_base64 shouldn't be empty with the placeholder, and the code will already raise before that point if there is another problem).